ports/elecdude/Elec Dude.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+
+XDG_DATA_HOME=${XDG_DATA_HOME:-$HOME/.local/share}
+
+if [ -d "/opt/system/Tools/PortMaster/" ]; then
+  controlfolder="/opt/system/Tools/PortMaster"
+elif [ -d "/opt/tools/PortMaster/" ]; then
+  controlfolder="/opt/tools/PortMaster"
+elif [ -d "$XDG_DATA_HOME/PortMaster/" ]; then
+  controlfolder="$XDG_DATA_HOME/PortMaster"
+else
+  controlfolder="/roms/ports/PortMaster"
+fi
+
+source $controlfolder/control.txt
+source $controlfolder/device_info.txt
+export PORT_32BIT="Y"
+
+get_controls
+[ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"
+
+$ESUDO chmod 666 /dev/tty0
+
+GAMEDIR="/$directory/ports/elecdude"
+
+export LD_LIBRARY_PATH="/usr/lib32:$GAMEDIR/libs"
+export GMLOADER_DEPTH_DISABLE=1
+export GMLOADER_SAVEDIR="$GAMEDIR/gamedata/"
+
+# We log the execution of the script into log.txt
+> "$GAMEDIR/log.txt" && exec > >(tee "$GAMEDIR/log.txt") 2>&1
+
+cd $GAMEDIR
+
+# check if we have new enough version of PortMaster that contains xdelta3
+if [ ! -f "$controlfolder/xdelta3" ]; then
+  echo "This port requires the latest PortMaster to run, please go to https://portmaster.games/ for more info." > /dev/tty0
+  sleep 5
+  exit 1
+fi
+
+# Patch game
+cd "$GAMEDIR"
+
+# If "gamedata/data.win" exists and matches the checksum of the itch or steam versions
+if [ -f "./gamedata/data.win" ]; then
+    checksum=$(md5sum "./gamedata/data.win" | awk '{print $1}')
+    
+    # Checksum for the itch.io version
+    if [ "$checksum" = "adc1dc81b4e8b1640f29714400333b62" ]; then
+        $ESUDO $controlfolder/xdelta3 -d -s gamedata/data.win -f ./patch/itch.xdelta gamedata/game.droid && \
+        rm gamedata/data.win
+    # Checksum for the Steam version
+    elif [ "$checksum" = "0e8fe70a1aca955a1db05a338f9ee6b7" ]; then
+        $ESUDO $controlfolder/xdelta3 -d -s gamedata/data.win -f ./patch/steam.xdelta gamedata/game.droid && \
+        rm gamedata/data.win
+    else
+        echo "Error: MD5 checksum of game.unx does not match any expected version."
+    fi
```

Comment:
Should the script exit here?
```suggestion
      exit 1
    fi
```

Reply:
Shall input this.